feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/node_config.c

```diff
@@ -362,11 +363,18 @@ static void led_handler(int type,
 				sscanf(sysled,"/%[^/]/%[^/]/%[^/]/%s", sys, class, leds, led_string);
 				sscanf(led_string,"%[^:]:%[^:]:%s",ap_name, color, led_section);
 				if(available_led_check(led_section)) {
+					snprintf(led_name_final, sizeof(led_name_final), "%s%s","led_",led_section);
 					if(!strcmp(conf->key, "led_blink")) {
-						set_led_config("heartbeat", conf->key, conf->value, led_string, led_section);
+						set_led_config("heartbeat", conf->key, conf->value, led_string, led_name_final);
 					}
 					else if(!strcmp(conf->key, "led_off")) {
-						set_led_config("none", conf->key, conf->value, led_string, led_section);
+						set_led_config("none", conf->key, conf->value, led_string, led_name_final);
+					}
+					else if(!strcmp(conf->key, "default_state")) {
+						if(!strcmp(led_section, "wifi2g"))
+							set_led_config("phy1tx", conf->key, conf->value, led_string, led_name_final);
```

Comment:
I am not sure wifi2g is phy1tx for all APs.  Similar for wifi5g is phy0tx.

For example, my ECW5211 has board.json has phy0 for wifi2g.
root@OpenAp-b906af:/etc# cat board.json 
{
	"model": {
		"id": "edgecore,ecw5211",
		"name": "Edgecore ECW5211"
	},
	"led": {
		"wlan2g": {
			"name": "WLAN2G",
			"sysfs": "ecw5211:green:wlan2g",
			"trigger": "phy0tpt"
		},
		"wlan5g": {
			"name": "WLAN5G",
			"sysfs": "ecw5211:green:wlan5g",
			"trigger": "phy1tpt"
		}
	},
	"network": {
		"lan": {
			"ifname": "eth1",
			"protocol": "static"
		},
		"wan": {
			"ifname": "eth0",
			"protocol": "dhcp"
		}
	}
}

Also, what about EA8300 which has 3 radios and so likely has a phy2tx as well?

Reply:
Yes, I'll fix this issue based on the board and EA8300 doesn't have 2G and 5G LEDs.

Reply:
done